### PR TITLE
Add Parameters to allowlist

### PR DESCRIPTION
**Pull request by Jasopaum**
## Description
Add torch.nn.Parameters to allowlist

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by iamtrask** [comment]
Looks like a small linting error. rerun black?

**Comment by iamtrask** [comment]
I'm excited about this one! Let's include some autograd tests if we can?

**Comment by Jasopaum** [comment]
Work merged with #4525
